src/installer/installer.go

```diff
@@ -410,6 +410,18 @@ func (i *installer) startBootstrap() error {
 		return err
 	}
 
+	// If we're in a pure RHEL/CentOS environment, we need to overlay the node image
+	// first to have access to e.g. oc, kubelet, cri-o, etc...
+	// https://github.com/openshift/enhancements/pull/1637
+	if !i.ops.FileExists("/usr/bin/oc") {
+		err = i.ops.SystemctlAction("start", "node-image-overlay.service")
+		if err != nil {
+			return err
+		}
+
+		i.waitForNodeImageOverlay(context.Background())
```

Comment:
Should there be a check performed after this wait to ensure that `/usr/bin/oc` is available as expected?